src/validators/relatives.js

```diff
@@ -159,6 +159,12 @@ export class RelativeValidator {
 
   validAliases() {
     const items = this.aliases.items || []
+    const nonImmediateFamily = ['Fosterparent', 'Father-in-law', 'Mother-in-law', 'Guardian']
```

Comment:
Looks good – I double-checked this list against the list on page 94 of [Validation Matrix SF 86 2017_08_07.pdf](https://drive.google.com/open?id=1_lIHjuKC2t8hNWOnKJDOAdF7Z7e8ylAI).